yolox/models/yolo_head.py

```diff
@@ -526,92 +516,49 @@ def get_assignments(
             num_fg,
         )
 
-    def get_in_boxes_info(
+    def get_geometry_constraint(
         self,
         gt_bboxes_per_image,
         expanded_strides,
         x_shifts,
         y_shifts,
-        total_num_anchors,
-        num_gt,
     ):
```

Comment:
IMHO, docstring is needed to explain the new function.

Reply:
done